core/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@snowbridge/core",
   "private": true,
   "engines": {
-    "node": "v18.16.0",
-    "pnpm": "8.4.0"
+    "node": "v18.16.1",
+    "pnpm": "8.6.6"
```

Comment:
We may also sync node&pnpm in github action.
https://github.com/Snowfork/snowbridge/blob/a9d3df4e065622bde119d4a8ef953a1b5a1c883e/.github/workflows/ethereum.yml#L23-L29